### DISCUSSION TITLE
广州大学（小谷围）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#22] [University]
> 回答来自群友，我仅负责搬运

**宿舍是上床下桌吗**
否。宿舍为老四人间改六人间，三张上下铺带六个桌子，桌面面积狭小。

**教室和宿舍有没有空调**
是。教室空调冷到头痛。宿舍空调效率不高且有时间限制（21:00 至次日 06:00；午 12:00 至 14:30）

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
是。部分宿舍卫浴无分离。卫浴质量看舍友质量。不会大扫除的宿舍污垢厚且有异味，多扫除就很干净。

**有早自习、晚自习吗**
是。大一有，基本上都翘了，没人管你

**有晨跑吗**
否。

**每学期跑步打卡的要求是多少公里，可以骑车吗**
男生 97km/term。女生要少一点。大一大二每学期都要。大三两学期一算。自由骑单车。

**寒暑假放多久，每年小学期有多长**
正常每学期 19 周左右。无小学期。

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
是。疫情前可送至楼下。疫情后新建了围墙，看你离围墙的远近程度。

**学校附近有地铁站吗**
否。广佛环线新站可步行至，但是在教学区距离较远且为城际轨道。地铁站需搭乘 15 分钟公交。

**宿舍楼有洗衣机吗**
否。可自行购买。

**校园网怎么样**
速度可以，连接不够稳定。我不打游戏，据打游戏的舍友说体验不佳，我自用还好。

**每天断电断网吗，几点开始断**
否、是。不断电。01:00 至次日 06:00 断网

**食堂价格贵吗，会吃出异物吗**
12-15 CNY 一餐。食物质量在破岛上垫底水平。可能吃出异物。（所以我都是去隔壁广美、广工、华师吃饭的）

**洗澡热水供应时间**
18:00 至 00:00

**校园内可以骑电瓶车吗，电池在哪能充电**
待补充
是。没骑过，不太清楚

附注：
大一广大在整个趋向于封闭的大学城中仍然是非常开放的存在，那时候校园里没有围墙，有几乎是开放式街区一样的各种通道，入口。晚间摆夜市的小贩也可以在生活区里自由穿梭。我有幸在大学的前两年体会过如此重要的自由，故而才为今日的这座筑满高墙的孤岛而扼腕叹息。如果为了功利，那广大适合适合它的你。如果为了自由，我只想说：快逃！  
另：欢迎了解 ingress。世界并非你所见的那样。

**Comment by Rachel030219** [comment]
来自群友的补充：

我想更正一下广州大学相关的情况

宿舍楼有洗衣机，可能不是每栋宿舍都有，投币的，但是大部分人估计不敢用的，太脏了
校园网是IPv4下载30Mbps(某些时候会达到50Mbps，可能是根据负载情况调整的)上传2Mbps，IPv6不限速(硬件只支持百兆宽带，所以可以认为是百兆限速)
(我觉得很有必要收集一下校园网是否支持IPv6，以及费用问题，高校PT站还是很香的，我两个舍友之前就是懒得搞PT，现在毕业了都注册了，都说真香)
上网费用30元/月，无限流量，包半年或包年有优惠(好像是包半年送一个月，包年送两个月？)
断网其实是封IPv4的80和443端口，以及会在任意端口上探测HTTP协议并封禁，这意味着如果你有非80 443端口的代理那还是能上网，且断网期间IPv4似乎不限速
食堂印象中没那么贵，看你选啥的，品种很多。而且广大有商业中心，里面也有一堆吃的，选择挺丰富的。旁边还有北亭村，也是一堆吃的，去隔壁学校吃饭这种做法我觉得非常奇怪。至于质量，我觉得和外卖差不多，异物没吃出过
电瓶车充电的地方在商中旁边，跟宿舍区隔开的，一般都是送外卖的车在充电，宿舍区好像没有充电的地方，所以可以认为学生是不能骑电瓶车的
至于是否自由这个主观性质较大，虽然说疫情爆发后确实封闭了许多，但是个人感觉相比别的学校还是挺好的，比如有商中，进出校门(非宿舍门)不需要刷卡(中山大学要)，校园网有IPv6(广工似乎没有)

**Reply by Rachel030219** [reply]
来自同一个群友的匿名补充：

有的人可能觉得我是护校蛆，只会说好话，但我觉得我只是要求不高罢了，有的人要求高有的人要求低，比如时不时就见人吐槽校园网网速慢，说实话上传2Mbps确实慢，但是下载30Mbps还不限流量感觉不错了，有的学校还是按流量计费的，这可能跟学校财力有关。目前我不爽的是空调没法全天开(但是空调完全免费，广工独立空调似乎是要自己交电费)。

另外标题最好改成大学城，你说小谷围没啥外地人知道的。

还有补充一个：

据说断网后连接学校提供的VPN也能上网，不过我没用过，速度如何也不知道，断网后我都直接开代理了。

**Reply by Rachel030219** [reply]
同样是群友的匿名补充：

还有新内容：

校园卡用的是汇银通公司的大学城一卡通。

热水需要用卡，热水钱包要在微信公众号上充值后去充值机圈存。

吃饭、消费的账号是招商银行的账号(III类账号)，跟你的卡绑定，存钱进去之后可以直接刷卡，不需要圈存。这个招行的账户可以绑定支付宝、微信使用，一般不用担心充多了花不完。

另外汇银通的公众号也提供微信支付充钱到招行账户的服务，不一定要别的银行卡转账进去。如果你微信里一堆钱想免手续费提现的话，可以用这个充进去，然后再用招行app转到别的银行卡。

还有，也很有必要收集一下校园网是否可以使用路由器。
广州大学可以，需要配合第三方认证软件。
